tools/export_to_path/export_to_path.py

```diff
@@ -15,38 +14,47 @@
 parser = optparse.OptionParser()
 parser.add_option('-d', '--export_dir', help='Directory where to export the datasets')
 (options, args) = parser.parse_args()
-if not options.export_dir:
-    parser.error('Export directory cannot be empty')
-
-dir_prefix = "/mnt/sally"
-dir_suffix = 'mzML_profile'
-full_export_dir = os.path.join(dir_prefix, options.export_dir.lstrip(os.sep), dir_suffix)
-real_export_dir = os.path.realpath(full_export_dir)
-if not real_export_dir.startswith(dir_prefix):
-    raise Exception("'%s' must be a subdirectory of '%s'" % (real_export_dir, dir_prefix))
-if not os.path.exists(real_export_dir):
-    os.makedirs(real_export_dir)
+
+
+def get_path():
+    dir_prefix = '/mnt/sally'
+    dir_suffix = 'mzML_profile'
+    if not options.export_dir:
+        parser.error('Input cannot be empty')
+    elif not os.path.isdir(os.path.join(dir_prefix, options.export_dir.lstrip(os.sep))):
+        parser.error('Subdirectory must exist')
+    else:
+        return os.path.join(dir_prefix, options.export_dir.lstrip(os.sep), dir_suffix)
+
+
+export_path = get_path()
+
+
+if not os.path.exists(export_path):
+    os.makedirs(export_path)
```

Comment:
Here you are creating the directory after you've already failed the script on the line 25 above. This is unreachable code.